pandas/__init__.py

```diff
@@ -190,6 +193,17 @@
     __git_version__ = v.get("full-revisionid")
     del get_versions, v
 
+# GH#55043 - deprecation of the data_manager option
+if "PANDAS_DATA_MANAGER" in os.environ:
```

Comment:
In theory, `PANDAS_DATA_MANAGER` could be set after pandas is imported. I'm guessing there's no nice way to detect that

Reply:
We check os.environ when the option is registered, which happens on import. Setting the environment variable after importing pandas has no effect.